svgdigitizer/models/exceptions.py

```diff
@@ -0,0 +1,101 @@
+r"""
+A collection of custom exceptions for svgdigitizer.
+"""
+# ********************************************************************
+#  This file is part of svgdigitizer.
+#
+#        Copyright (C) 2021-2023 Albert Engstfeld
+#
+#  svgdigitizer is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  svgdigitizer is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with svgdigitizer. If not, see <https://www.gnu.org/licenses/>.
+# ********************************************************************
+
+
+class SVGContentError(RuntimeError):
+    """Raised when facing issues with the structure of the SVG input."""
+
+
+class AnnotationError(SVGContentError):
+    """The annotations in the SVG are is missing or incorrect."""
+
+class CurveError(SVGContentError):
```

Comment:
I would not create these super-specialized exceptions. It's just too much work to maintain this code I fear.

Reply:
So you mean simply 

```python
class SomeError(RuntimeError):
    """Some description"""
```

And then call the Error via
```python
raise SomeError("Error Message")
```

?

Reply:
Yes.